dev/Scroller/ScrollerPrimitives.idl

```diff
@@ -37,7 +37,7 @@ unsealed runtimeclass SnapPointBase
 unsealed runtimeclass ScrollSnapPointBase
     : SnapPointBase
 {
-    ScrollSnapPointsAlignment Alignment { get; set; };
```

Comment:
This is an interesting decision, I'm curious what the rational was.

Reply:
- All other properties are read-only: Value, Start, End, Interval, Offset.
- Supporting a setter would require the ScrollSnapPointBase class to have a callback mechanism so all owning Scrollers are aware when its Alignment property changes & regenerate everything.